docs/setup/Connecting.md

```diff
@@ -0,0 +1,45 @@
+---
+sidebar_position: 7
+---
+
+# Connecting to the Server
+
+## Launcher (Recommended)
+1. Download [MS2 Launcher](https://github.com/Clockworkx/MS2-Launcher/releases/download/v1.0/MS2.Launcher.v1.0.1.7z).
+2. Extract the launcher to a folder.
+3. Make a shortcut of MS2Launcher.Exe (rightclick->Send to->Desktop)
+4. Start the Launcher.
+5. You can configure your IPs using the disk-pen symbol on the bottom-right of the launcher. Change your IP and Port and click on the save icon.
+6. Select whether you want to play Local (using the server on your PC) or Online (the remote server configured in step 5).
+7. Click Start. On first lauunch the Launcher will ask you to select your MapleStory2.exe.
+8. Enjoy.
+
+* Note: You can see the online status of the server in the top-right corner of the corresponding server.
+
+## Manual
+
+If your server is running on your PC you can proceed with Connecting to Localhost. If you are playing on a server that is NOT on your computer, go to the Connecting to Remote Server section.
+
+## Connecting to Localhost
+
+1. If you have NOT done any changes to your server port, you are ready to play: Doubleclick the MapleStory2.exe which is located in Your_MapleStoryClient_Installation_Path\appdata\MapleStory2.exe.
+   
+2. You can select any server. The id and password that you enter will be used to CREATE an account, once you click on login. So make sure to remember your data. 
+
+3. You can rightclick -> Send to -> Desktop (create shortcut), so you can start Maple Story from your Desktop, next time.
+
+4. If you have NOT done any changes to your server ports, you are ready to play. Enjoy. 
```

Comment:
I think this can be removed, already mentioned on step 1

Reply:
agreed. I changed the structure!